src/otel.ts

```diff
@@ -49,6 +49,13 @@ const otel129: OtelVersion = {
     [ColumnHint.TraceTags, 'SpanAttributes'],
     [ColumnHint.TraceServiceTags, 'ResourceAttributes'],
     [ColumnHint.TraceStatusCode, 'StatusCode'],
+    [ColumnHint.TraceKind, 'SpanKind'],
+    [ColumnHint.TraceStatusMessage, 'StatusMessage'],
+    [ColumnHint.TraceInstrumentationLibraryName, 'InstrumentationLibraryName'],
```

Comment:
I noticed theres a few columns defined in `otel.ts` that don't exist on the [ClickHouse OTel exporter default schema](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/eac16d39fcf5d201894b9babae3eff79dce4e58c/exporter/clickhouseexporter/example/default_ddl/traces.sql). The intent of these definitions is to match as closely as possible to this table schema for a better out of the box experience.

Should we look into adding these columns to the otel exporter?

Also note that `State` is defined too, but on the table it is called `TraceState`.